changes/en-us/2.x.md

```diff
@@ -50,9 +50,9 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6147](https://github.com/apache/incubator-seata/pull/6147)] upgrade kafka-clients to 3.6.1
 
 ### test:
-- [[#6081](https://github.com/apache/incubator-seata/pull/6081)] add `test-os.yml` for testing the OS
-- [[#PR_NO](https://github.com/apache/incubator-seata/pull/PR_NO)] A brief and accurate description of PR
-- [[#6125](https://github.com/apache/incubator-seata/pull/6125)] unbind xid in TransactionTemplateTest
+- [[#6081](https://github.com/seata/seata/pull/6081)] add `test-os.yml` for testing the OS
```

Comment:
I think need revert this line, and all seata/seata should change to apache/incubator-seata.